cli/azd/cmd/util.go

```diff
@@ -290,7 +290,8 @@ func ensureEnvironmentInitialized(
 }
 
 func getSubscriptionOptions(ctx context.Context, azCli azcli.AzCli) ([]string, any, error) {
-	accountManager, err := account.NewManager(config.GetConfigManager(ctx), azCli)
+	var accountManager *account.Manager
+	err := ioc.Global.Resolve(&accountManager)
```

Comment:
We should avoid using the container directly. Instead the account manager should be passed in as a dependency or be able to be constructed like it was prior to this PR.